models/web/scratch/interactions_this_run/snowflake/snowplow_media_player_interactions_this_run.sql

```diff
@@ -55,7 +57,7 @@ with prep as (
     {% elif var("snowplow__enable_whatwg_media") %}
       e.contexts_org_whatwg_media_element_1[0]:htmlId::varchar as media_id,
       'org.whatwg-media_element' as media_player_type,
-      e.contexts_org_whatwg_media_element_1[0]:currentSource::varchar as source_url,
+      e.contexts_org_whatwg_media_element_1[0]:currentSrc::varchar as source_url,
```

Comment:
What happened here?

Reply:
I recognised while running the integration test that this field retrieved null for snowflake due to the wrong name. It is really not a seemingly really useful field or anything but it needs correcting. Thank god you highlighted this as I checked it again and realised I still had it wrong for the integration tests and it did not come up as I was testing with all media player variables enabled where I still had it wrong.. You will see everything changed to currentSrc which it should be.